src/pygeoprocessing/geoprocessing_core.pyx

```diff
@@ -868,8 +869,8 @@ def _raster_band_percentile_double(
     raster_info = pygeoprocessing.get_raster_info(
         base_raster_path_band[0])
     nodata = raster_info['nodata'][base_raster_path_band[1]-1]
-    n_pixels = numpy.prod(raster_info['raster_size'])
-    pixels_processed = 0
+    cdef long long n_pixels = numpy.prod(raster_info['raster_size'])
```

Comment:
I think the tuple we're getting and putting into `prod` isn't doing 64bit operations.
```python
>>> numpy.prod((129600, 64800))
-191854592
>>> array = numpy.array([129600, 64800], dtype=numpy.float64)
>>> array
array([129600.,  64800.])
>>> numpy.prod(array)
8398080000.0
```

Reply:
ahh got it, fixed in c5e0db8